static/docs/commands-reference/repro.md

```diff
@@ -186,3 +187,42 @@ Saving information to 'Dvcfile'.
 
 You can check now that `Dvcfile` and `count.txt` have been updated with the new
 information, new `md5` checksums and a new result respectively.
+
+There is also an option which allows one to reproduce results from a specific
```

Comment:
let's create a separate section `## Examples: Downstream` or something like that. Check `dvc add` to see an example of this split. It easier to navigate, read, and make sense of example.